### PR TITLE
Group Merchant Items by Status

### PR DESCRIPTION
**Pull request by SullyBirashk**
- Added Two Sections to Merchant Items Index Page
- One is labeled Enabled Items
- The Other is labeled Disabled Items
- Each section shows the right item, with a button to change the status

### CODE REVIEW

File: app/views/merchant_items/index.html.erb [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-<% if item.status == "Disabled" %>
-<%= button_to "Enable #{item.name}", "/items/#{item.id}?status=enable", method: :patch %>
-<% elsif item.status == "Enabled" %>
-<%= button_to "Disable #{item.name}", "/items/#{item.id}?status=disable", method: :patch %>
-<% end %>
-<% end %>
+<div class="enabled_items">
+  <ol>
+    <h2>Enabled Items:</h2>
+    <% @merchant.items.each do |item| %>
```
Comment (AustinCMoore) [inline app/views/merchant_items/index.html.erb:7]: Not going to make you do it on this one but next user story consider making this more readable

File: spec/features/merchants/items/index_spec.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@
     @merchant_1 = Merchant.create!(name: "Staples")
     @item_1 = @merchant_1.items.create!(name: "stapler", description: "Staples papers together", unit_price: 13)
     @item_2 = @merchant_1.items.create!(name: "paper", description: "construction", unit_price: 29)
+    @enabled_item = @merchant_1.items.create!(name: "pen", description: "red ink", unit_price: 5, status: "Enabled")
```
Comment (AustinCMoore) [inline spec/features/merchants/items/index_spec.rb:9]: Can we move this out of the before block and into the test you need it for? I think it makes it more readable for this test, but may mess us up in the future.

**Review by AustinCMoore — CHANGES_REQUESTED** [review]
Can you move enabled item into your "it" block?

**Comment by SullyBirashk** [comment]
Got these Changes resolved!

**Pull request merged**